client/app/components/_pickupList/pickupList.spec.js

```diff
@@ -0,0 +1,294 @@
+import PickupListModule from "./pickupList";
+import UserModule from "../../common/user/user";
+import StoreModule from "../../common/store/store";
+import AuthenticationModule from "../../common/authentication/authentication";
+import PickupDateModule from "../../common/pickupDate/pickupDate";
+
+describe("PickupList", () => {
+  let $rootScope, $componentController, $httpBackend;
+
+  let { module } = angular.mock;
+
+  beforeEach(module(PickupListModule));
+  beforeEach(module(UserModule));
+  beforeEach(module(StoreModule));
+  beforeEach(module(AuthenticationModule));
+  beforeEach(module(PickupDateModule));
+
+  beforeEach(inject(($injector) => {
+    $httpBackend = $injector.get("$httpBackend");
```

Comment:
Each time you use $httpBackend, check if there are no outstanding & requests:
```
  afterEach(() => {
    $httpBackend.verifyNoOutstandingExpectation();
    $httpBackend.verifyNoOutstandingRequest();
  });
```
I'll add this here :)